Importer/CsvFileImporter.php

```diff
@@ -11,7 +14,8 @@ class CsvFileImporter extends CsvImporter
     public function import($filename, $dryRun = false)
     {
         if (!file_exists($filename)) {
-            $this->log->addConfigurationError('File not found');
+            $error = new FileNotFoundError();
+            $this->eventDispatcher->dispatch('EVENT_CONFIGURATION_ERROR', new ConfigurationErrorEvent($error, $this->log));
```

Comment:
It also seems a bit strange to pack the error object inside the event object. could we just use one object?